runtime/src/iree/vm/BUILD.bazel

```diff
@@ -78,6 +78,7 @@ iree_runtime_cc_library(
     ],
     deps = [
         "//runtime/src/iree/base",
+        "//runtime/src/iree/base:core_headers",
```

Comment:
base pulls in what's needed - you can remove the #include of alignment.h and this dep